### DISCUSSION TITLE
Contact Velero's feature

### DISCUSSION THREAD

**Discussion by asigga** [vmware-tanzu/velero#5928] [Community support Q&A]
I had several problems installing and configuring with Velero. 
And I'm still working on it, but I have a question.
I want to know how to check options like --restic-timeout, --default-restic-prune-frequency. 
I wonder if there is a way to search for it, or if there is a site to organize it.
Thanks.

**Comment by asigga** [comment]
I I checked through the command velero help server.
But It is curious that there is no information about the server when the velero help command is executed.
Thanks

**Comment by shubham-pampattiwar** [comment]
`velero help server` command should help you with flags that can be set during install time, if you want to see tet values set for the server flag for your velero installation then you might have to check the velero deployment pod's container args.
